### PR TITLE
Add ability to strip whitespace from tool environment_variables.

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

**Comment by martenson** [comment]
@blankenberg The failing tests seem relevant.

**Comment by blankenberg** [comment]
> @blankenberg The failing tests seem relevant.

Agreed, thanks for the ping :)

File: lib/galaxy/tool_util/xsd/galaxy.xsd [hunk 1]
```diff
@@ -4583,6 +4583,11 @@ variable instead of shell variable.
 define.</xs:documentation>
           </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="strip" type="PermissiveBoolean" default="false">
+        <xs:annotation>
+          <xs:documentation xml:lang="en">Whether to strip whitespace.</xs:documentation>
```
Comment (nsoranzo) [inline lib/galaxy/tool_util/xsd/galaxy.xsd]:
```suggestion
          <xs:documentation xml:lang="en">Whether to strip leading and trailing whitespace from the calculated value before exporting the environment variable.</xs:documentation>
```

**Review by nsoranzo — APPROVED** [review]
Thanks @blankenberg !

It would be nice to have some unit tests for `<environment_variables/>` in future.

**Pull request merged**